### PR TITLE
implemented xml

### PR DESCRIPTION
**Pull request by mlgubolin**

### CODE REVIEW

File: lib/dyzn_web/router.ex [hunk 1]
```diff
@@ -13,10 +13,10 @@ defmodule DyznWeb.Router do
     plug :accepts, ["json"]
   end
 
-  scope "/", DyznWeb do
+  scope "/rest", DyznWeb do
```
Comment (umamaistempo) [inline lib/dyzn_web/router.ex:16]: Suggestion: `"/api"

File: lib/dyzn_web/router.ex [hunk 2]
```diff
@@ -13,10 +13,10 @@ defmodule DyznWeb.Router do
...
     pipe_through :browser
 
-    get "/", PageController, :index
+    get "/", PageController, :ping
```
Comment (umamaistempo) [inline lib/dyzn_web/router.ex:19]: Suggestion: `"/ping"`

File: lib/dyzn_web/views/page_view.ex [hunk 3]
```diff
@@ -1,3 +1,9 @@
 defmodule DyznWeb.PageView do
   use DyznWeb, :view
+
+  def render("index.html",params) do
+    params.xml
+    |> Phoenix.HTML.raw()
+    # |> Phoenix.HTML.safe_to_string()
```
Comment (umamaistempo) [inline lib/dyzn_web/views/page_view.ex:7]: Forgot to remove this

File: lib/dyzn_web/views/page_view.ex [hunk 4]
```diff
@@ -1,3 +1,9 @@
 defmodule DyznWeb.PageView do
   use DyznWeb, :view
+
+  def render("index.html",params) do
+    params.xml
+    |> Phoenix.HTML.raw()
```
Comment (umamaistempo) [inline lib/dyzn_web/views/page_view.ex:6]: Smart. Just note that you are appending `conn.query_params["u"]` on `lib/dyzn_web/controllers/page_controller.ex` which can be a security risk.

Maybe you can return to the view the different parts of the body and it decides what is safe and what is not.

Eg:
```elixir
    conn
    |> Plug.Conn.put_resp_content_type("application/xml")
    |> render("index.html", xml_header: ..., unsafe_body: ..., xml_footer: ...)
```

File: mix.exs [hunk 5]
```diff
@@ -10,7 +10,8 @@ defmodule Dyzn.MixProject do
       compilers: [:phoenix, :gettext] ++ Mix.compilers(),
       start_permanent: Mix.env() == :prod,
       aliases: aliases(),
-      deps: deps()
+      deps: deps(),
+      dir: "/Users/mlgubolin/Music"
```
Comment (umamaistempo) [inline mix.exs:14]: ?

File: lib/dyzn_web/router.ex (hunk 1 above)
Comment (mlgubolin) [reply lib/dyzn_web/router.ex:16]: tem que ser /rest pq o subsonic faz assim ._.

File: lib/dyzn_web/router.ex (hunk 2 above)
Comment (mlgubolin) [reply lib/dyzn_web/router.ex:19]: mesma coisa

File: mix.exs (hunk 5 above)
Comment (mlgubolin) [reply mix.exs:14]: Removing.

**Pull request merged**